includes/admin/dashboard-widgets.php

```diff
@@ -42,6 +42,33 @@ function edd_dashboard_sales_widget( ) {
 	echo '<p><img src=" ' . esc_attr( set_url_scheme( EDD_PLUGIN_URL . 'assets/images/loading.gif', 'relative' ) ) . '"/></p>';
 }
 
+/**
+ * Get the sales earnings/count data for the dashboard widget.
```

Comment:
WordPress tries to stick with _third-person singular verbs_ for summaries: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#language

So something like `Gets` or `Retrieves` would be more appropriate here.